packages/main/src/components/Splitter/index.tsx

```diff
@@ -343,6 +368,15 @@ const Splitter = forwardRef((props: SplitterPropTypes, ref: Ref<HTMLDivElement>)
     setIsSiblings(isRtl && !vertical ? ['nextSibling', 'previousSibling'] : ['previousSibling', 'nextSibling']);
   }, [isRtl, vertical]);
 
+  const currentTheme = useCurrentTheme();
+  const isHighContrast =
+    currentTheme == 'sap_fiori_3_hcb' ||
+    currentTheme == 'sap_fiori_3_hcw' ||
+    currentTheme == 'sap_horizon_hcb' ||
+    currentTheme == 'sap_horizon_hcw'
+      ? true
+      : false;
```

Comment:
You can omit the ternary operator as the result will be `true` or `false` anyways. Also even if it shouldn't matter in this case, you should only use the equality operator (`==`) if you really need that behavior. In all other cases the strict equality operator should be used (`===`). The difference between these two is that `==` converts different types, so e.g. `0 == "0"` would be `true`. https://dorey.github.io/JavaScript-Equality-Table/
```suggestion
  const isHighContrast =
    currentTheme === 'sap_fiori_3_hcb' ||
    currentTheme === 'sap_fiori_3_hcw' ||
    currentTheme === 'sap_horizon_hcb' ||
    currentTheme === 'sap_horizon_hcw';
```